packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -474,11 +474,14 @@ abstract class FlutterCommand extends Command<void> {
       );
   }
 
-  void addNullSafetyModeOptions() {
+  void addNullSafetyModeOptions({ @required bool hide }) {
     argParser.addFlag(FlutterOptions.kNullSafety,
-      help: 'Whether to override the default null safety setting.',
+      help:
+        'Whether to override the inferred null safety mode to support out of '
+        'order migrations. By default, Flutter applications will attempt to '
```

Comment:
out-of-order has hyphens as an adjective.

What does "out of order" mean exactly?

Reply:
Rephrased